### PR TITLE
camunda-operate: Add version 1.2.0

### PR DESCRIPTION
**Pull request by jfastnacht**

### CODE REVIEW

File: bucket/camunda-operate.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "extract_dir": "camunda-operate-$matchVersion"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/zeebe-io/zeebe/releases/download/$matchRelease/camunda-operate-$matchVersion.zip"
+            }
+        }
+    }
+}
```
Comment (Ash258) [inline bucket/camunda-operate.json]:
```suggestion
{
    "version": "1.2.0",
    "description": " Workflow Engine for Microservices Orchestration",
    "homepage": "https://zeebe.io",
    "license": {
        "identifier": "Zeebe Community License Version 1.0",
        "url": "https://zeebe.io/legal/zeebe-community-license-1.0/"
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/zeebe-io/zeebe/releases/download/0.22.1/camunda-operate-1.2.0.zip",
            "hash": "sha1:b8e6330d5e2c917250add386b8077524fa74c54e"
        }
    },
    "extract_dir": "camunda-operate-1.2.0",
    "bin": [
        [
            "bin\\operate.bat",
            "comunda-operate"
        ]
    ],
    "checkver": {
        "github": "https://github.com/zeebe-io/zeebe",
        "regex": "download/(?<release>[\\d.]+)/camunda-operate-([\\d.]+)\\.zip"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/zeebe-io/zeebe/releases/download/$matchRelease/camunda-operate-$version.zip"
            }
        },
        "hash": {
            "url": "$url.sha1sum"
        },
        "extract_dir": "camunda-operate-$version"
    }
}
```

Comment (jfastnacht) [reply bucket/camunda-operate.json]: It says "Workflow Engine for Microservices Orchestration" on the GitHub page, but it's actually the tool to "Monitor, manage and troubleshoot running workflow instances". I don't know why they decided to distribute both tools within one repository.

Camunda Cloud

**Comment by jfastnacht** [comment]
> What is needed? From clean shim it is not working

It's missing the Elastic instance to connect to

**Comment by Ash258** [comment]
So why did you not add suggests for elastic and java, which are obviously required?

File: bucket/camunda-operate.json (hunk 1 above)
Comment (Ash258) [reply bucket/camunda-operate.json]: Then apply these changes and find the correct homepage.

Comment (jfastnacht) [reply bucket/camunda-operate.json]: Camunda Cloud is an SaaS version of Zeebe + Operate (+ Cawemo + Optimize).

Operate is called Camunda Operate right now, even though it is in the Zeebe (which is a Camunda product as well) repository. It's similar to the Camunda Cockpit (https://camunda.com/products/cockpit/), but it's not the same.

IMHO it's either https://zeebe.io / https://docs.zeebe.io/operate-user-guide/index.html or nothing. There is no dedicated page for Operate.

File: bucket/camunda-operate.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+    "version": "1.2.0",
+    "description": "Camunda workflow instance monitor/manager",
+    "homepage": "https://github.com/zeebe-io/zeebe",
+    "license": {
+        "identifier": "Zeebe Community License Version 1.0",
+        "url": "https://zeebe.io/legal/zeebe-community-license-1.0/"
+    },
+    "architecture": {
```
Comment (Ash258) [inline bucket/camunda-operate.json]:
```suggestion
    "suggest": {
        "JDK": [
            "java/oraclejdk",
            "java/openjdk"
        ],
        "ElasticSearch": [
            "extras/elasticsearch"
        ]
    },
    "architecture": {
```

File: bucket/camunda-operate.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    ],
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ],
+        "ElasticSearch": [
+            "extras/elasticsearch"
+        ]
+    },
```
Comment (Ash258) [inline bucket/camunda-operate.json]:
```suggestion
```

File: bucket/camunda-operate.json [hunk 4]
```diff
@@ -1,27 +1,46 @@
 {
-    "homepage": "https://github.com/zeebe-io/zeebe",
     "version": "1.2.0",
-    "license": "Zeebe Community License Version 1.0",
-    "description": "Monitor, manage and troubleshoot running workflow instances.",
+    "description": "Camunda workflow instance monitor/manager",
+    "homepage": "https://github.com/zeebe-io/zeebe",
```
Comment (Ash258) [inline bucket/camunda-operate.json]:
```suggestion
    "homepage": "https://docs.zeebe.io/operate-user-guide/",
```

**Comment by Ash258** [comment]
Why `config` folder is not persisted?

**Comment by jfastnacht** [comment]
> Why `config` folder is not persisted?

Thanks for the advice, I haven't thought about it.

File: bucket/camunda-operate.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    "extract_dir": "camunda-operate-1.2.0",
+    "bin": [
+        [
+            "bin\\operate.bat",
+            "camunda-operate"
+        ]
+    ],
+    "persist": [
+        "config"
+    ],
```
Comment (Ash258) [inline bucket/camunda-operate.json]:
```suggestion
    "persist": "config",
```

**Pull request closed without merging**